Sources/Build/Command.compile(ClangModule).swift

```diff
@@ -105,6 +105,7 @@ extension Command {
         var args = module.basicArgs
         args += module.optimizationFlags(conf)
         args += ["-L\(prefix)"]
+        args += module.languageArgs
```

Comment:
This isn't going to have the desired effect. Technically, if we wanted to do the right thing we would need to look at all dependencies of each ultimate link command, and see if any used C++, and if so then DTRT.